app/controllers/hosts_controller.rb

```diff
@@ -936,6 +936,9 @@ def host_attributes_for_templates(host)
     # should be ignored by Host::Discovered or any other Host class that does
     # not have that attribute
     host_attributes = host.class.attribute_names
+    if Foreman::Cast.to_bool(params[:host][:managed])
+      host_attributes << 'compute_attributes'
```

Comment:
This would cause the `compute_attributes` item to be added to `Host.class.attribute_names` every time we call `host_attributes_for_templates`. You need to use `dup`

Reply:
That was the key - thanks @iNecas ! 🤕